idea-plugin/build.gradle

```diff
@@ -28,17 +28,14 @@ intellij {
 patchPluginXml {
     pluginDescription = "Formats source code using the palantir-java-format tool."
     version = project.version
-    sinceBuild = '173' // TODO: test against this version of IntelliJ to ensure no regressions
+    sinceBuild = '182' // TODO: test against this version of IntelliJ to ensure no regressions
     untilBuild = ''
 }
 
 publishPlugin {
     token = System.env.JETBRAINS_PLUGIN_REPO_TOKEN
-    if (!versionDetails().isCleanTag) {
-        channels 'eap'
-    }
```

Comment:
this was pointless, and I'm not even sure how I could consume the snapshots if I wanted to (even tho i'm subscribed to the EAP plugin list)